tests/route/test_forced_mgmt_route.py

```diff
@@ -107,43 +127,50 @@ def test_forced_mgmt_route_add_and_remove_by_mgmt_port_status(
     override_config["MGMT_INTERFACE"] = config_db_mgmt_interface
     override_config["MGMT_PORT"] = config_db_port
     logging.debug("override_config: {}".format(override_config))
-    reload_minigraph_with_golden_config(duthost, override_config)
+    wait_for_file_changed(
+                        duthost,
+                        "/etc/network/interfaces",
+                        reload_minigraph_with_golden_config,
+                        duthost,
+                        override_config)
 
     # Get interface and check config generate correct
     interfaces = duthost.command("cat /etc/network/interfaces")['stdout']
     logging.debug("interfaces: {}".format(interfaces))
     pytest_assert("iface eth1 inet static" in interfaces)
-    pytest_assert("up ip -4 rule add pref 32764 to 172.17.1.1/24 table default" in interfaces)
-    pytest_assert("pre-down ip -4 rule delete pref 32764 to 172.17.1.1/24 table default" in interfaces)
+    pytest_assert("up ip -4 rule add pref 32764 to {} table default"
```

Comment:
>32764 [](http://example.com/codeflow?start=42&length=5)

magic number is repeated many times, we need to have at least named constant in this test file.

Reply:
Fixed, add code link as code comments also create new constant.